062-quotas-management.md

```diff
@@ -0,0 +1,163 @@
+# Quotas management
+
+This proposal introduces a new mechanism for managing the Apache Kafka quota plugins.
+It supports both the Apache Kafka's built-in Quota plugin as well as the new (not yet released) version of the [Strimzi Quota plugin](https://github.com/strimzi/kafka-quotas-plugin).
+
+## Current situation
+
+### Quotas in Apache Kafka
+
+Apache Kafka supports a pluggable quota mechanism that allows to limit the load Kafka users and/or clients can put on the Kafka brokers.
+
+Apache Kafka provides a built-in plugin that is enabled by default and can apply quotas to:
+* Maximum bytes per-second that each client group can publish to a broker
+* Maximum bytes per-second that each client group can fetch from a broker
+* Maximum CPU utilization as a percentage of network and I/O threads
+* Rate at which mutations are accepted for the create topics request, the create partitions request and the delete topics request
+
+These quotas can be set at the user or client level and are useful to:
+* Prevent a single client from overloading the broker
+* Prevent noisy neighbor situations, where one very busy user or client negatively impacts others
+* Ensure fairness in allocating resources to different users and clients
+
+The built-in quota plugin is enabled when `client.quota.callback.class` is set to `null` (i.e. not set) in Apache KAfka configuration.
+
+Instead of using the default plugin, users can also write and use their own quota plugins by implementing the `ClientQuotaCallback` Java interface and configuring it with the `client.quota.callback.class` in Kafka configuration.
+The custom implementations can have different goals.
+For example the [Strimzi Quota plugin](https://github.com/strimzi/kafka-quotas-plugin) focus on preventing the Kafka brokers from running out of disk space instead of limiting the individual clients or users.
```

Comment:
```suggestion
For example, the [Strimzi Quota plugin](https://github.com/strimzi/kafka-quotas-plugin) focuses on preventing the Kafka brokers from running out of disk space instead of limiting the individual clients or users.
```